app/src/main/java/com/asterisk/tuandao/themoviedb/data/source/model/Movie.kt

```diff
@@ -1,39 +1,60 @@
 package com.asterisk.tuandao.themoviedb.data.source.model
 
-import com.asterisk.tuandao.themoviedb.data.source.model.respone.GenreResponse
+import com.asterisk.tuandao.themoviedb.data.source.model.respone.CastResult
+import com.asterisk.tuandao.themoviedb.data.source.model.respone.VideoResult
+import com.google.gson.annotations.Expose
 import com.google.gson.annotations.SerializedName
 
 class Movie {
     @SerializedName("id")
+    @Expose
```

Comment:
có biết thêm Expose để làm gì hông?

Reply:
em thêm expose vào đây để convert string to object ạ